cli/internal/packager/deploy.go

```diff
@@ -178,3 +185,69 @@ func deployComponents(tempPath componentPaths, assets config.ZarfComponent) {
 		git.PushAllDirectories(tempPath.repos)
 	}
 }
+
+// If privded package is a URL download it to a temp directory
```

Comment:
"privded" :-)

Reply:
Spelling is one of my biggest weaknesses.. I just downloaded a spell checking extension for VSCode :P